pandas/core/generic.py

```diff
@@ -11176,6 +11179,8 @@ def _doc_params(cls):
 -------
 {name1} or {name2} (if level specified) \
 {notes}
+
+{examples}
```

Comment:
Will there be extra whitespace at the end for e.g. `sem` (where you have `examples=""`) now? Could you post a screenshot of those docs too please?

Reply:
Will double-check and send a screenshot shortly